bots/images/scripts/lib/kubevirt.setup

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -ex
+
+TEMP_DIR="/tmp/kubevirt.build"
+rm -rf ${TEMP_DIR}
+mkdir -p ${TEMP_DIR}
+cd ${TEMP_DIR}
+
+# download latest Kubevirt release
+KUBEVIRT_TGZ=`curl -s https://api.github.com/repos/kubevirt/kubevirt/releases/latest | grep \"tarball_url\": | sed -e 's/^.*\"tarball_url\":.*\"\(.*\)\".*$/\1/'`
+echo Latest Kubevirt TGZ: ${KUBEVIRT_TGZ}
+
+# FIXME: Kubevirt v0.1.0 is not yet released but will contain required #602 pull request
+echo XXXXXXXX Fix me: rewriting to pre-release:
+KUBEVIRT_TGZ="https://api.github.com/repos/kubevirt/kubevirt/tarball/v0.1.0-alpha"
```

Comment:
@mareklibra could you have a look at https://github.com/kubevirt/user-guide/pull/7? It contains instructions on how to install kubevirt based on the v0.1.0-alpha release. You can pull the manifests from the pre-release: https://github.com/kubevirt/kubevirt/releases/download/v0.1.0-alpha/kubevirt.yaml.

Reply:
Done